contracts/strategies/traderJoe/strategy-joe-avax-vso.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+import "../strategy-joe-farm-base.sol";
+
+contract StrategyJoeAvaxVsoLp is StrategyJoeFarmBase {
+
+    uint256 public avax_vso_poolId = 11;
+
+    address public joe_avax_vso_lp = 0x00979Bd14bD5Eb5c456c5478d3BF4b6E9212bA7d;
+    address public vso = 0x846D50248BAf8b7ceAA9d9B53BFd12d7D7FBB25a;
+
+    constructor(
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+        public
+        StrategyJoeFarmBase(
+            avax_vso_poolId,
+            joe_avax_vso_lp,
+            _governance,
+            _strategist,
+            _controller,
+            _timelock
+        )
+    {}
+
+    // **** State Mutations ****
+
+    function harvest() public override onlyBenevolent {
+        // Anyone can harvest it at any given time.
+        // I understand the possibility of being frontrun
+        // But AVAX is a dark forest, and I wanna see how this plays out
+        // i.e. will be be heavily frontrunned?
+        // if so, a new strategy will be deployed.
+
+        // Collects Joe tokens
+        IMasterChefJoeV2(masterChefJoeV2).deposit(poolId, 0);
+
+        uint256 _joe = IERC20(joe).balanceOf(address(this));
+        if (_joe > 0) {
+            // 10% is sent to treasury
+            uint256 _keepJOE = _joe.mul(keepJOE).div(keepJOEMax);
+            IERC20(joe).safeTransfer(
+                IController(controller).treasury(),
+                _keepJOE
+            );
+            uint256 _amount = _joe.sub(_keepJOE).div(2);
+            IERC20(joe).safeApprove(joeRouter, 0);
+            IERC20(joe).safeApprove(joeRouter, _joe.sub(_keepJOE));
+
+            _swapTraderJoe(joe, wavax, _amount);
+            _swapTraderJoe(joe, vso, _amount);
+        }
+
+        // Collects VSO tokens for double reward
```

Comment:
Yeah, this is about how I thought it would be implemented. Very straight forward and elegant. Good job. So if it isn't being provided, not a big deal. Definitely something to consider updating on the other contracts with traderJoe just in case they get double rewards at some point. Definitely need to mark this one somewhere as a reference file.